lib/lotus/generators/action.rb

```diff
@@ -108,6 +114,14 @@ def assert_action!
         end
       end
 
+      # @since x.x.x
+      # @api private
+      def assert_http_method!
+        if !HttpRouter::Route::VALID_HTTP_VERBS.include?(_http_method.upcase)
```

Comment:
Done.  Not sure if it is better now, had to require it. Left it as a single commit so far, can squash if you like it.

Now, since apparently the existing code uses _verb_ and _method_ (but more _verb_) i was wondering if i should rename everything in my PR from method to verb?